build.py

```diff
@@ -4,5 +4,21 @@
 if __name__ == "__main__":
     builder = ConanMultiPackager()
     builder.add_common_builds(shared_option_name="gtest:shared", pure_c=False)
+    filtered_builds = []
+    for settings, options in builder.builds:
+        if settings["compiler"] == "Visual Studio":
+            pdbOptions = options.copy()
+            noPdbOptions = options.copy()
+            
+            pdbOptions.update({"gtest:include_pdbs": "True"})
+            noPdbOptions.update({"gtest:include_pdbs": "False"})
```

Comment:
Done, see: [122460d](https://github.com/lasote/conan-gtest/pull/10/commits/122460d863d64eb89a96cf5be5890e1dfcee7032). I removed the redundant option and simplified the branch condition in the loop.